src/diff/children.js

```diff
@@ -69,38 +69,25 @@ export function diffChildren(
 		// or we are rendering a component (e.g. setState) copy the oldVNodes so it can have
 		// it's own DOM & etc. pointers
 		else if (typeof childVNode == 'string' || typeof childVNode == 'number') {
-			childVNode = newParentVNode._children[i] = createVNode(
-				null,
-				childVNode,
-				null,
-				null,
+			childVNode = newParentVNode._children[i] = createBackingNode(
```

Comment:
We shouldn't always recreate the backing node, we'd have to reuse these from the `oldChildren`